packages/babel-preset-default/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/babel-preset-default",
-	"version": "1.0.0",
+	"version": "1.0.1-0",
```

Comment:
@gziolo `1.0.1-0`? Was this meant to be `1.0.1`?

Reply:
I wanted to check what happens when i reference this preset from the `package.json` in the root folder. Unfortunately when I do it, it doesn't use the latest code, rather published version from the npm. I will continue using relative path to `./packages/babel-preset-default`. I will publish stable version as `1.0.0`. This was pre-release tag.

Reply:
Right 👍, `.babelrc` isn't taking precedence over `package.json` is it?

Reply:
I don’t remember, they explain it in their docs 😃